common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js

```diff
@@ -0,0 +1,24 @@
+try {
+    ((document, window) => {
+        var ad = document.createElement('div');
+        ad.style.position = 'absolute';
+        ad.style.left = '-9999px';
+        ad.style.height = '10px';
+        ad.innerHTML = '&nbsp;';
+        ad.setAttribute('class', 'ad_unit');
+
+        window.requestAnimationFrame(() => {
+            document.body.appendChild(ad);
```

Comment:
I know this should be fine in practice, but it does feel a bit weird to have this running in a loosely asynchronous fashion.